src/dawn/CodeGen/GridTools/GTCodeGen.cpp

```diff
@@ -186,6 +195,61 @@ class StencilFunctionAsBCGenerator : public ASTCodeGenCXX {
   }
 };
 
+std::string GTCodeGen::cacheWindowToString(boost::optional<Cache::window> const& cacheWindow) {
+  DAWN_ASSERT(cacheWindow.is_initialized());
+  return std::string("window<") + std::to_string((*cacheWindow).m_m) + "," +
+         std::to_string((*cacheWindow).m_p) + ">";
+}
+
+std::string GTCodeGen::buildMakeComputation(std::vector<std::string> const& DomainMapPlaceholders,
```

Comment:
yes, I agree. 
const T is more widely used. The reason why I like the other is for readability when using pointers. 
The fact that `const T *` and `T const*` (pointer to constant T) have the same semantic but completely different than `T * const` (constant pointer to T) is very confusing and not consistent. 
But if you stick to `T const *` and read from right to left is always consistent. 

But I am ok with the more usual convention `const T *`

Reply:
I've seen both sides of the argument. For me, `const T` feels more natural (most likely because it is widely used) and I think most of our code base is already in that style. I'm open to both solutions though